backend/src/service/loggerService.js

```diff
@@ -0,0 +1,53 @@
+const winston = require('winston')
```

Comment:
I don't think Winston is better than Morgan, I've even seen that there are developers that used them together, so both must have their advantages, in this case I must to say I used winston because it is the one I allways, use and  know